docs/content/tardigrade.md

```diff
@@ -0,0 +1,306 @@
+passphrase> your-human-readable-encryption-passphrase
+Remote config
+--------------------
+[remote]
+type = tardigrade
+satellite_address = 12EayRS2V1kEsWESU9QMRseFhdxYxKicsiFmxrsLZHeLUtdps3S@us-central-1.tardigrade.io:7777
+api_key = your-api-key-for-your-tardigrade-project
+passphrase = your-human-readable-encryption-passphrase
+access_grant = the-access-grant-generated-from-the-api-key-and-passphrase
+--------------------
+y) Yes this is OK (default)
+e) Edit this remote
+d) Delete this remote
+y/e/d> y
+```
+
+## Usage
+
+Paths are specified as `remote:bucket` (or `remote:` for the `lsf`
+command.)  You may put subdirectories in too, eg `remote:bucket/path/to/dir`.
+
+Once configured you can then use `rclone` like this.
+
+### Create a new bucket
+
+Use the `mkdir` command to create new bucket, e.g. `bucket`.
+
+    rclone mkdir remote:bucket
+
+### List all buckets
+
+Use the `lsf` command to list all buckets.
+
+    rclone lsf remote:
+
+Note the colon (`:`) character at the end of the command line.
+
+### Delete a bucket
+
+Use the `rmdir` command to delete an empty bucket.
+
+    rclone rmdir remote:bucket
+
+Use the `purge` command to delete a non-empty bucket with all its content.
+
+    rclone purge remote:bucket
+
+### Upload objects
+
+Use the `copy` command to upload an object.
+
+    rclone copy --progress /home/local/directory/file.ext remote:bucket/path/to/dir/
+
+The `--progress` flag is for displaying progress information.
+Remove it if you don't need this information.
+
+Use a folder in the local path to upload all its objects.
+
+    rclone copy --progress /home/local/directory/ remote:bucket/path/to/dir/
+
+Only modified files will be copied.
+
+### List objects
+
+Use the `ls` command to list recursively all objects in a bucket.
+
+    rclone ls remote:bucket
+
+Add the folder to the remote path to list recursively all objects in this folder.
+
+    rclone ls remote:bucket/path/to/dir/
+
+Use the `lsf` command to list non-recursively all objects in a bucket or a folder.
+
+    rclone lsf remote:bucket/path/to/dir/
+
+### Download objects
+
+Use the `copy` command to download an object.
+
+    rclone copy --progress remote:bucket/path/to/dir/file.ext /home/local/directory/
+
+The `--progress` flag is for displaying progress information.
+Remove it if you don't need this information.
+
+Use a folder in the remote path to download all its objects.
+
+    rclone copy --progress remote:bucket/path/to/dir/ /home/local/directory/
+
+### Delete objects
+
+Use the `deletefile` command to delete a single object.
+
+    rclone deletefile remote:bucket/path/to/dir/file.ext
+
+Use the `delete` command to delete all object in a folder.
+
+    rclone delete remote:bucket/path/to/dir/
+
+### Print the total size of objects
+
+Use the `size` command to print the total size of objects in a bucket or a folder.
+
+    rclone size remote:bucket/path/to/dir/
+
+Sync `/home/local/directory` to the remote bucket, deleting any excess
```

Comment:
This sentence doesn't fit `Print the total size of objects` paragraph.

Reply:
True. Forgot to remove it. Removed now.